### ISSUE TITLE
Problems in training binding predicition model.

### ISSUE THREAD

**Issue by KiAkize** [s6juncheng/BERTMHC#8]
* BERTMHC version: 0.1.0
* Python version: 3.7.11
* Operating System: linux
* Non-commercial research [Y/N]: Y

**Q1:** In the cli.py file, `aff_criterion = nn.BCEWithLogitsLoss()`. However, the paper said `We use the standard mean squared error loss function for the binding model and the weighted cross-entropy loss function (weight 10 for the positive class) for the presentation model when training with SA data.`  The code does not match the description of the paper, how did you train the model to get the results on the BA data?

**Q2:** In five-fold cross-validation and LOMO experiments on BA data, the same data as NetMHCIIpan3.2, what data should be set as --eval, respectively? What metic should be set as --metric, respectively?
